libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -196,7 +193,7 @@ Example:
 
 `unique`:: (Optional) Defaults to `false`. Marking an autodiscover provider as unique results into
   making the provider to enable the provided templates only when it will gain the leader lease.
-  This setting can only be combined with `cluster` scope. When `unique` is enabled enabled, `resource`
+  This setting can only be combined with `cluster` scope. When `unique` is enabled, `resource`
```

Comment:
Not related with this PR but what do we mean when we say here that with unique: true  the `add_resource_metadata` settings are not taken into account?

Reply:
If `unique: true` then we are not using watchers, I think that's why they are not taken into account.